sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -623,6 +623,39 @@ def patch_item(
             response_hook(self.client_connection.last_response_headers, result)
         return result
 
+    @distributed_trace
+    def bulk(
+        self,
+        operations: List[Dict[str, Any]],
+        **kwargs: Any
+    ) -> Dict[str, Any]:
+        """ **Provisional method** Bulk execute the list of operations on the given items.
+        Can be used for create, upsert, read, replace and delete operations.
+
+        :param operations: The list of bulk operations to execute.
+        :type operations: List[Dict[str, Any]]
+        :keyword str pre_trigger_include: trigger id to be used as pre operation trigger.
```

Comment:
Do we absolutely need triggers for bulk?

Reply:
every request in the Python SDK exposes triggers to users, it's part of the Python SDK designs by Core team